zcash-issue-dag.py

```diff
@@ -298,23 +303,74 @@ class Meta:
 
     def __init__(self, *args, **kw):
         super(ZenHubAPI, self).__init__(*args, **kw)
-        self.add_resource('dependencies', ZHDepsResourceHandler)
         self.add_resource('epics', ZHEpicsResourceHandler)
         self.add_resource('epics_issues', ZHEpicsIssuesResourceHandler)
 
     def auth(self, *args, **kw):
         pass
 
+def fetch_workspace_graph(op, workspace_id, repos, cursor):
+    dependencies = op.workspace(id=workspace_id).issue_dependencies(
+        # TODO: This causes a 500 Internal Server Error.
+        # repository_ids=repos,
```

Comment:
IDK why this causes a 500 (AFAICT I'm setting the argument with correctly-formatted data), but it means that we're always fetching the edges for every repository in the workspace, not just the workspaces we care about. This slows down rendering significantly compared to the REST API.

Someone should contact ZenHub to inquire about this bug.

Reply:
Oh you came to the same conclusion.